app/src/main/java/com/example/tour_a_versity_v3/MapsActivity.kt

```diff
@@ -127,36 +123,24 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnInfoWi
         )
 
     }
+
+    fun createCechMarker() {
+        val cechLat = 39.130315
+        val cechLng = -84.518680
+        val cechLatLng = LatLng(cechLat, cechLng)
+        cechMarker = map.addMarker(MarkerOptions()
+            .position(cechLatLng)
+            .title("CECH")
+            .snippet("College of Education, Criminal Justice, Human Services, and IT")
+        )
+    }
```

Comment:
I recommend abstracting out the specifics into private methods so that your `onMapReady` method doesn't get too long. Also, optimally this method should really consume a data class instead of having data hardcoded into it. Unfortunately this change requires more code changes than how much I can do in the scope of a code review, so I cannot make those changes for you and add them here.

Reply:
These changes would have been fine for a code review.  Easier to understand something you can see than something you read about.